_im-plugin/index-codecs.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: default
+title: Index codecs
+nav_order: 3
+parent: Index settings
+---
+
+# Index codecs
+
+Index codecs determine how the index’s stored fields are compressed and stored on disk. It is controlled by the static `index.codec` setting that specifies the compression algorithm. The setting impacts the index shard size and index operation performance.  
```

Comment:
Second sentence: "They are" instead of "It is"?

Reply:
Reworded.